tests/requirements.txt

```diff
@@ -0,0 +1,27 @@
+bok_choy==0.3.2
```

Comment:
@e-kolpakov @Kelketek @bradenmacdonald The `tests/requirements.txt` file was a result of `pip freeze`. I removed `bock_choy` and the tests still work, but it looks like the other dependencies are all needed for the SDK to load. The `XBlock` repository is pinned to the version currently used by `edx-solutions/edx-platform`.

I really really like to pin down the dependencies for testing. Note that these are not in the main `requirements.txt` file - you would only install the `tests/requirements.txt` dependencies for testing, preferably in a disposable virtualenv, so there really shouldn't be any conflicts.
I believe having a simple way to create a consistent environment for testing is very important. I hate wasting time manually installing dependencies just to be able to run the tests.